docs/make.jl

```diff
@@ -11,12 +11,12 @@ makedocs(;
     authors="Rémi Vezy <VEZY@users.noreply.github.com> and contributors",
     repo=Documenter.Remotes.GitHub("VirtualPlantLab", "PlantSimEngine.jl"),
     sitename="PlantSimEngine.jl",
-    format=Documenter.HTML(;
-        prettyurls=get(ENV, "CI", "false") == "true",
-        canonical="https://VirtualPlantLab.github.io/PlantSimEngine.jl",
-        edit_link="main",
-        assets=String[],
-        size_threshold=300000
+    format=DocumenterVitepress.MarkdownVitepress(;
+        deploy_url="https://VirtualPlantLab.github.io/PlantSimEngine.jl",
+        repo="github.com/VirtualPlantLab/PlantSimEngine.jl",
```

Comment:
Might need a `https`
```suggestion
        repo="https://github.com/VirtualPlantLab/PlantSimEngine.jl",